examples/dynamic-dao-implementation/README_pt.adoc

```diff
@@ -0,0 +1,379 @@
+:index-group: Proxy Beans
+:jbake-type: page
+:jbake-status: status=published
+= Implementação dinâmica de DAO
+
+Muitos aspectos dos Data Access Objects (DAOs) são muito repetitivos e
+chapa de ebulição. Como um recurso divertido e experimental, o TomEE suporta
+implementar dinamicamente uma interface que é vista como tendo padrão
+Métodos no estilo DAO.
+
+A interface deve ser anotada com @PersistenceContext para definir
+qual EntityManager usar.
+
+Os métodos devem respeitar estas convenções:
+
+* void save(Foo foo): persistir foo
+* Foo update(Foo foo): merge foo
+* void delete(Foo foo): remover foo, se foo for desconectado, ele tenta
+anexá-lo.
+* Collection|Foo namedQuery(String name[, Map<String, ?> params, int
+first, int max]): executar a consulta nomeada chamada name, params contém
+ligações, first e max são usadas para paginação. Últimos três parâmetros
+são opcionais.
+* Collection|Foo nativeQuery(String name[, Map<String, ?> params, int
+first, int max]): executar a consulta nativa chamada name, params contém
+ligações, first e max são usadas para paginação. Últimos três parâmetros
+são opcionais.
+* Collection|Foo query(String value [, Map<String, ?> params, int first,
+int max]): executar a consulta colocada como primeiro parâmetro, params contém
+ligações, first e max são usadas para paginação. Últimos três parâmetros
+são opcionais.
+* Collection findAll([int first, int max]): Encontre todos os Foo, os parâmetros são
+usado para paginação.
+* Collection findByBar1AndBar2AndBar3(<bar 1 type> bar1, <bar 2 type>
+bar2, bar3 [, int first, int max]): encontre todos os Foo com campo especificado
+valores para bar1, bar2, bar3.
+
+O localizador dinâmico pode ter o quanto você deseja restrições de campo. Para
```

Comment:
"pode ter quantas constantes de campo você quiser." instead of "o quanto você deseja restrições de campo".